### DISCUSSION TITLE
1차 리젝 공유 및 대처방안 회의

### DISCUSSION THREAD

**Discussion by Haroldfromk** [SwiftBreakers/TTEOPPOKKI4U#68] [Ideas]
![CleanShot 2024-06-19 at 05 26 05@2x](https://github.com/SwiftBreakers/TTEOPPOKKI4U/assets/97341336/d9787454-8adb-495d-a361-8b6e9dc10824)

1차 배포시도에서 다음과 같은 리젝이 발생 하였고, 계정 베이스로 운영하지말고 유져가 자유롭게 Access 할 수 있도록 앱을 수정하라는 방침입니다.

해당 부분을 자유롭게 이야기 해보면서 해결 방안을 찾고 다시 재배포를 해야할것같습니다.

만약 계정 베이스로 가려는 방침이라면 회신이 필요해보입니다.

![CleanShot 2024-06-19 at 05 27 58@2x](https://github.com/SwiftBreakers/TTEOPPOKKI4U/assets/97341336/b46070ce-eb50-486d-a962-28866f215212)

**Comment by Haroldfromk** [comment]
저는 지금 생각을 한게
1. 게스트 모드의 UIbutton을 하나 만들어서 지도, 리뷰확인만 가능하게 구성.
2. 리젝사유에 대해 저희가 계정기반으로 운영 할 수밖에 없는 이유를 적어 회신

우선은 떠오르는건 이 2가지 정도인것같습니다.

**Reply by jinmoon23** [reply]
추천과 지도는 로그인 하지 않아도 접근가능하도록 해야겠네요. 
북마크 버튼 누르면 로그인하지 않은 유저는 로그인 페이지로 이동하도록 구성하면 좋을 것 같습니다.

**Comment by Haroldfromk** [comment]
1차 대처 내용.
---

**게스트로그인 버튼을 추가하면서 해당 리젝에 대한 문제 해결**
게스트 로그인을 추가하면서 보완한 기능.
1. 추천페이지
- 북마크 쪽 기능을 유저 로그인 제한으로 Deactive
2. 지도페이지
- 게스트로 로그인시 Bottom Sheet가 활성화가 안되는 문제가 있어 해결
- 스크랩 기능을 유저 로그인 제한으로 Deactive
3. 리뷰페이지
- 리뷰작성 페이지, 신고기능을 유저 로그인 제한으로 Deactive
4. 마이페이지
- 프로필 수정, 설정 페이지 Deactive
